pages/draft-06/json-hyper-schema-release-notes.md

```diff
@@ -1,107 +1,104 @@
+| `"schema"`            | _removed_             | use `"hrefSchema"`, `"submissionSchema"`, or `"targetSchema"`                                                                                                                                                                                                                                                                         |
+| `"encType"`           | _removed_             | use `"submissionEncType"` for request bodies; no longer needed for URI query strings                                                                                                                                                                                                                                                  |
+| `"hrefSchema"`        | _added_               | replaces `"method": "get", "schema": {...}`, with additional functionality                                                                                                                                                                                                                                                            |
+| `"submissionSchema"`  | _added_               | replaces `"method": "post", "schema": {...}`                                                                                                                                                                                                                                                                                          |
+| `"submissionEncType"` | _added_               | replaces `"method": "post", "encType": "..."`                                                                                                                                                                                                                                                                                         |
+| `"href"`              | preprocessing removed | _to be replaced and expanded in future drafts_                                                                                                                                                                                                                                                                                        |
 
-#### Proper use of `"targetSchema"`
+##### Proper use of `"targetSchema"`
 
-While `"targetSchema"` did not change its meaning in either recent draft, it has been widely misinterpreted.  So it may feel like a change to use it as specified.
+While `"targetSchema"` did not change its meaning in either recent draft, it has been widely misinterpreted. So it may feel like a change to use it as specified.
```

Comment:
suggestion: change this sentence to avoid writing in the negative for readability. 

Example: "Although 'targetSchema' has kept its meaning, its interpretation has been misinterpreted. Therefore, using it as intended might feel like a modification."

Reply:
Done.